tests/utils.go

```diff
@@ -4551,15 +4598,14 @@ func IsRunningOnKindInfra() bool {
 	return strings.HasPrefix(provider, "kind")
 }
 
-func SkipPVCTestIfRunnigOnKindInfra() {
-	if IsRunningOnKindInfra() {
-		Skip("Skip PVC tests till PR https://github.com/kubevirt/kubevirt/pull/3171 is merged")
-	}
+func IsRunningOnKindInfraIPv6() bool {
+	provider := os.Getenv("KUBEVIRT_PROVIDER")
+	return strings.HasPrefix(provider, "kind-k8s-1.17.0-ipv6")
```

Comment:
I don't understand this; isn't the issue that kind (the only infra where we can currently run ipv6 tests on) is docker on docker, and as such we get the same machine id on all k8s nodes ? Why add a new function about ipv6 on kind, instead of re-using https://github.com/kubevirt/kubevirt/pull/3221/files#diff-a5a178b4c66835110ceff63c61d28c96R4596 ?

Reply:
since its not in the same func